### ISSUE TITLE
Support ellipsis_mask in stridedslice_op?

### ISSUE THREAD

**Issue by jiafatom** [onnx/tensorflow-onnx#367]
Ellipsis mask is commonly used in tensorflow, but currently stridedslice op does not support ellipsis mask. Can we add this support? Thanks.

**Comment by guschmue** [comment]
which model ran into it ?

**Comment by jiafatom** [comment]
@guschmue , yolov3 model uses that https://github.com/qqwweee/keras-yolo3

**Comment by jiafatom** [comment]
See https://github.com/qqwweee/keras-yolo3/blob/master/yolo3/model.py
def yolo_correct_boxes(box_xy, box_wh, input_shape, image_shape):
    box_yx = box_xy[..., ::-1]
    box_hw = box_wh[..., ::-1]

    boxes =  K.concatenate([
        box_mins[..., 0:1],  # y_min
        box_mins[..., 1:2],  # x_min
        box_maxes[..., 0:1],  # y_max
        box_maxes[..., 1:2]  # x_max
    ])

**Comment by guschmue** [comment]
got it, first model we run into that needs ellipsis mask. If I remember this correctly the existing Slice op in onnx did not allow this but will look again at it.

**Comment by guschmue** [comment]
fixed in master
